Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/http_profile_model.ts

```diff
@@ -30,6 +30,22 @@ export class HttpProfileModel extends ProfileModel {
     return this.model.identity;
   }
 
+  public get isReadonly(): boolean {

```

Comment:
This could work but in the future propose changes to the API before submitting them in a PR.